i18n-helpers/Cargo.toml

```
@@ -17,12 +17,12 @@ anyhow.workspace = true
 chrono = { version = "0.4.33", default-features = false, features = ["alloc"] }
 mdbook.workspace = true
 polib.workspace = true
-pulldown-cmark = { version = "0.9.2", default-features = false }
+pulldown-cmark = { version = "0.10.0", default-features = false }
```

Let's put this back: this is actually a breaking release and @max-heller is already working on updating pulldown-cmark-to-cmark to handle the breakage: https://github.com/Byron/pulldown-cmark-to-cmark/pull/67.

```suggestion
pulldown-cmark = { version = "0.9.2", default-features = false }
```